### ISSUE TITLE
Purge duplicate requests in FOI Mod

### ISSUE THREAD

**Issue by lmullane** [bcgov/foi-flow#3600]
Title of ticket: 

#### Description
Summarize issue 

At times requests have been opened in Mod before they were synced. Need to delete these duplicate requests.

https://bcgov.sharepoint.com/:x:/r/teams/03521/Shared%20Documents/General/Admin/Requests%20to%20Delete%20-%20FOI%20Mod.xlsx?d=w7f22a3d2a0fb4fa989f078cba47289c1&csf=1&web=1

#### Dependencies
Are there any dependencies?

#### DOD
- [ ] List the items that need to be complete for this ticket to be considered done
- [ ] 
- [ ] 
- [ ] 
- [ ]

**Comment by lmullane** [comment]
@m-prodan, do we have a more up to date purge list?

**Comment by m-prodan** [comment]
Yes - full list is here: https://bcgov.sharepoint.com/:x:/t/03521/EdKjIn_7oKlPifB4y6RyicEBsbj02J6X-SwUtcsc1NoE-A?e=Pbpm5L

**Comment by m-prodan** [comment]
Also seeing this might be up for refinement - I wonder if this is better suited as an update to the admin screen, or a function/UI we bring forward to Intake team? Would need designs, which I wouldn't prioritize over Jackie's other work right now.

**Comment by m-prodan** [comment]
will do the manual task of deleting the requests, but need to do an analysis first to determine impact to data tables and other issues when something is deleted.

**Comment by m-prodan** [comment]
Per discussion at refinement - we will purge the list, but also create a validation to ensure duplicates can no longer be created.

**Comment by KyEggleston** [comment]
Updated link/location so Intake can update directly. - https://bcgov.sharepoint.com/:x:/r/teams/05009/Shared%20Documents/General/Requests%20to%20Delete%20-%20FOI%20Mod.xlsx?d=w392c65bbea924d20a3dbef1836d2fbd9&csf=1&web=1&e=A8ijxj
